.github/workflows/buildtest.yml

```diff
@@ -29,3 +29,25 @@ jobs:
 
       - name: Go test
         run: sudo go test ./... # sudo needed for netns change in test
+
+  coverage:
+    runs-on: ubuntu-latest
+    needs: build
+    name: coverage
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16.x
```

Comment:
should we add a matrix to test with multiple go versions? WDYT?

Reply:
as we bumped to 1.18 in PR #209 
I think we should set here to 1.18.x 

also dont think it matters to run coverage with multiple go version.

Reply:
Adding a matrix to test multiple go versions involves another step to merge the coverage results, and it would make things a little bit more complicated.

The matrix is already in place for testing jobs, and I think code coverage doesn't differ too much between go versions.

But I just realized it's using v1.16.x, while testing jobs have been bumped to v1.17 and v.1.18. Going to fix it, thanks!